packages/angular_devkit/build_webpack/builders.json

```diff
@@ -1,12 +1,26 @@
 {
   "$schema": "../architect/src/builders-schema.json",
   "builders": {
+    "build": {
+      "implementation": "./src/webpack/index2",
```

Comment:
Is the idea here that transitioning to the new Architect API means adding the `implementation` field to existing builders, keeping both current and future version at the same time, then remove `class`/`schema` when ready to changeover?

Reply:
Yes, that's the general idea. Since we will remove the old API though, this will not show up past the beta period. The old API will be completely deleted before we move to RC.